### DISCUSSION TITLE
Didn’t arrive my XCH from exchanges to Chia Wallet app

### DISCUSSION THREAD

**Discussion by Finitossssss** [Chia-Network/chia-blockchain#14353] [Support]
So, I have a problem that I can't solve in any way. I installed the Chia Wallet for the first time and decided to transfer my Chia tokens from the Kukoin exchange to it. The transaction was completed on the exchange https://xchscan.com/zh/txns/0x4e24eeceaafab5c6e6702f1262bda30937154352bb8b2c310794be5debe6b7f2

But my Chia tokens are not displayed in my Chia Wallet, and there can be no errors in the address of the wallet or token network, I checked everything myself several times. The version of the Chia Wallet application is the latest.
Chia Wallet address is xch1sfuz02qlxk78gy2gwawdm5u4m364tmlqndx0hqudx9tzmxuudp7swqjhsp

How is this possible? What is the cause of the problem?

If you need any other clarification information, please ask!

**Comment by Bodman1** [comment]
good afternoon, I had a problem, I sent chia tokens from the exchange to the chia wallet, these tokens did not come to my wallet, the wallet address was correct, the tokens were sent from the exchange, now there are no tokens either on the exchange or on the wallet. what should I do and to whom should I write?
Chia Wallet address xch1ruzf693r9q8u2ldn8t9hh7adn446srf270gpdp5ydkstf76zgcgss4yd72

**Reply by shaneo257** [reply]
> good afternoon, I had a problem, I sent chia tokens from the exchange to the chia wallet, these tokens did not come to my wallet, the wallet address was correct, the tokens were sent from the exchange, now there are no tokens either on the exchange or on the wallet. what should I do and to whom should I write? Chia Wallet address xch1ruzf693r9q8u2ldn8t9hh7adn446srf270gpdp5ydkstf76zgcgss4yd72

per https://xchscan.com/address/xch1ruzf693r9q8u2ldn8t9hh7adn446srf270gpdp5ydkstf76zgcgss4yd72 it shows an inbound tx of approx 33xch

**Reply by Bodman1** [reply]
> > good afternoon, I had a problem, I sent chia tokens from the exchange to the chia wallet, these tokens did not come to my wallet, the wallet address was correct, the tokens were sent from the exchange, now there are no tokens either on the exchange or on the wallet. what should I do and to whom should I write? Chia Wallet address xch1ruzf693r9q8u2ldn8t9hh7adn446srf270gpdp5ydkstf76zgcgss4yd72
> 
> per https://xchscan.com/address/xch1ruzf693r9q8u2ldn8t9hh7adn446srf270gpdp5ydkstf76zgcgss4yd72 it shows an inbound tx of approx 33xch

this transaction was made on January 18, 2023, 7:22 am, at 33 chia, that evening I also transferred 27.9482 chia kucoin from the exchange to my wallet, in the support of the exchange they wrote to me that the coins were sent to the wallet, but on the wallet I didn't see them. transaction data

The withdrawal you made on 2023-01-19 03:04:26 (UTC+08:00) was successful.
Token: | XCH
Amount: | 27.9482
Address: | xch1ruzf693r9q8u2ldn8t9hh7adn446srf270gpdp5ydkstf76zgcgss4yd72

**Comment by shaneo257** [comment]
> So, I have a problem that I can't solve in any way. I installed the Chia Wallet for the first time and decided to transfer my Chia tokens from the Kukoin exchange to it. The transaction was completed on the exchange https://xchscan.com/zh/txns/0x4e24eeceaafab5c6e6702f1262bda30937154352bb8b2c310794be5debe6b7f2
> 
> But my Chia tokens are not displayed in my Chia Wallet, and there can be no errors in the address of the wallet or token network, I checked everything myself several times. The version of the Chia Wallet application is the latest. Chia Wallet address is xch1sfuz02qlxk78gy2gwawdm5u4m364tmlqndx0hqudx9tzmxuudp7swqjhsp
> 
> How is this possible? What is the cause of the problem?
> 
> If you need any other clarification information, please ask!

per https://xchscan.com/address/xch1sfuz02qlxk78gy2gwawdm5u4m364tmlqndx0hqudx9tzmxuudp7swqjhsp it shows an incoming tx of approx 9xch to the address 2 days ago
